keep-ui/app/mapping/rules-table.tsx

```diff
@@ -95,6 +95,11 @@ export default function RulesTable({ mappings, editCallback }: Props) {
         </div>
       ),
     }),
+    columnHelper.display({
+      id: "updated at",
+      header: "Updated At",
+      cell: (context) => context.row.original.last_updated_at,
+    }),
```

Comment:
Perhaps, if you want, you can check out what I did in Extractions (where you can "open" the row and see this details there)

Reply:
@pehlicd wdyt about this one? if you chose not to, maybe also add edited by?

Reply:
@talboren I just don't know what to check about it bro, could you share more insights about it?

Reply:
I updated it like extraction table fyi @talboren